### DISCUSSION TITLE
Check my config for export from only ever embylive - for later import only ever to embybackup

### DISCUSSION THREAD

**Discussion by vw-kombi** [arabcoders/watchstate#372] [General] [answered]
I am answering the setup questions for the embylive backend - this is my primary system.  I only ever want to run a manual export from this, then I want to run a manual import to the embybackup backend.

I guess import and export are reversed in cas eyou are thinking from the emby side or from the watchstate database side.

Is this correct setup then for the emby live - i.e I onlty want it to export my watchstates from embylive to your system and never the reverse.

I thought I was answering the questions correctly but the summary - I am not so sure.......

embylive:
  type: emby
  url: 'http://192.168.1.200:8096'
  token: xxxx
  uuid: xxxxx
  user: xxxxx
  import:
    enabled: true
  export:
    enabled: false
  webhook:
    match:
      user: false
      uuid: false
    token: xxxxx

**Comment by vw-kombi** [comment]
I think I amswered my own question - took an emby backup just in case - and alowed this config, then let it do an import from embylive, then setup the embybackup backend with import no, and then allowed it to do the metadata import from there only.  Fingers crossed.....

**Comment by vw-kombi** [comment]
OK - got it all working from the command line un unraid.  My issue now is getting it to be run by the scheduler userscript task on unraid that also sync;s the media files........

I have this in there to do the update of playstates from embylive backend - 
docker exec -ti WatchState console state:import -v --select-backends 'embylive'

But this is thrown in the script output - 
the input device is not a TTY

**Comment by vw-kombi** [comment, marked as answer]
So - the answer it to remove the 'terminal mode' - I removed the '-ti' on the command and ran it from the command prompts and it worked fine - so did that on the script.   Onwards and upwards......

**Reply by arabcoders** [reply]
Glad everything working for you!

**Comment by twinkie0101** [comment]
@vw-kombi - would appreciate helping out another unraid user that is struggling with the commands; no luck what i try:

all by accessing CLI via watchstate on docker tab:

$ docker exec -ti watchstate console config:add plex (-ti error)
$ docker exec watchstate console config:add plex (tried this one based on the comments above, but got bash: exec: watchstate: not found error)

using the server CLI access:
$ docker exec -ti watchstate console config:add plex (-ti error)
$ docker exec watchstate console config:add plex (command not found)

I am lost on how to get this working. Can you share the commands you used and any other tidbits of insight to help?

**Reply by vw-kombi** [reply]
The doco has a typo for unraid and maybe other dockers.  It’s case sensitive.  So WatchState.  Hope that helps.

**Reply by arabcoders** [reply]
Hi, Thanks for the information, sadly i dont have unraid so i cannot test it out, i didn't realize that unraid does not normalize the containers name, I pushed update to the templates to be lowercase thus bringing unraid containers inline with the docs.

Anyway, in unraid you should be accessing the container shell and using `console ......` directly, no need for `docker exec -ti WatchState` part as you are already inside the container =)

**Comment by twinkie0101** [comment]
Thank you @arabcoders - your guidance did the trick, able to set up watchstate now, thank you for your help. I am still new to linux, docker, etc. so appreciate the help! Cheers!
